packages/mdc-toolbar/README.md

```diff
@@ -0,0 +1,111 @@
+# MDC Toolbar
+
+MDC Toolbar provides an `mdc-toolbar` class for easily styling a RTL-aware
+toolbar. The toolbar scrolls with content by default, but supports fixed on
+top as well.
+
+
+## Installation
+
+```
+npm install --save @material/toolbar
+```
+
+
+## Usage
+
+Wrap the items with `mdc-toolbar` class in following way:
+
+```html
+<div class="mdc-toolbar">
+  <div class="mdc-toolbar__section">...</div>
+</div>
+```
+
+### Fix to top
+
+By default, toolbar scroll with the content, you can add `mdc-toolbar_fixed` to
+fix toolbar to the top of screen.
+
+**Body margin required**
+
+You need to wrap contents with `mdc-toolbar_fixed-container-adjust`
+when you apply `mdc-toolbar_fixed`, otherwise toolbar will overlay your other
+contents. There is no padding applied to this adjust class, so feel free to add
+your own padding.
+
+```html
+<div class="mdc-toolbar_fixed-container-adjust">
+  <div class="mdc-toolbar mdc-toolbar_fixed">
+  ...
+  </div>
+</div>
+```
+
+### Sections
+
+By default, the item aligns to the center of the toolbar. You can change the
+behavior by applying `mdc-toolbar__section--align-start` or
+`mdc-toolbar__section--align-end` to align it to the beginning or ending of the
+line.
+
+```html
+<div class="mdc-toolbar">
+  <div class="mdc-toolbar__section mdc-toolbar__section--align-start">
+  Item aligns to start.
```

Comment:
nit: Instead of saying "Item" here, should we say "Section" to make it less ambiguous?